<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-79CNQ9D" failures="1" tests="4" name="Test" time="0.164" errors="0" timestamp="2022-06-07T21:20:18 BST">
  <testcase classname="com.testng.Demo" name="password" time="0.025">
    <skipped/>
  </testcase> <!-- password -->
  <testcase classname="com.testng.Demo" name="password" time="0.003">
    <skipped/>
  </testcase> <!-- password -->
  <testcase classname="com.testng.Demo" name="password" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [123] but found [456]">
      <![CDATA[java.lang.AssertionError: expected [123] but found [456]
at com.testng.Demo.password(Demo.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- password -->
  <testcase classname="com.testng.Demo" name="username" time="0.003"/>
</testsuite> <!-- Test -->
